packages/shared/src/hooks/useMutateFilters.ts

```diff
@@ -59,6 +63,12 @@ async function updateQueryData(
       });
     }),
   );
+  if (alerts.filter) {
+    await request(`${apiUrl}/graphql`, UPDATE_ALERTS, {
+      data: { filter: false },
+    });
+    setAlerts({ ...alerts, filter: false });
+  }
```

Comment:
Here we dispatch again the update alerts request. We should have a single place that dispatches this call and then the other places may use this function/hook.